### DISCUSSION TITLE
pre-release of package for testing?

### DISCUSSION THREAD

**Discussion by Danfro** [atom/atom#23443] [Q&A]
snap atom 1.57.0

Hi, I am maintaining a package for atom. I am currently preparing a new release and I wonder if there is a way to provide a test package, a pre-release for testing. The [flight-manual](https://flight-manual.atom.io/hacking-atom/sections/publishing/) only contains information about publishing the actual release.

How can I provide a test package for others to make sure everything is working fine before the actual release?

**Comment by icecream17** [comment]
You can install your package locally by running `apm install` (or `apm-beta install`) at your project directory; then `apm link`

**Reply by Danfro** [reply]
Thanks for the reply. Currently I am editing the package code `~/.atom/packages/...`. This allows me to live test on my machine. Your suggestion would be another option for that.

But I still don't know how I can provide the package to others for testing. Like a .deb file I could post somewhere. Debs or snaps allow beta/dev installs.

Although based on your "-beta" keyword, a search provided me with some instructions how to run atom in dev mode and link packages to dev [https://flight-manual.atom.io/hacking-atom/sections/creating-a-theme/#creating-a-ui-theme](https://flight-manual.atom.io/hacking-atom/sections/creating-a-theme/#creating-a-ui-theme). This might be worth a try, although sounds not user friendly for testers.
